FreeRTOS_RA.c

```diff
@@ -71,6 +71,37 @@
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Find a link-local address that is bound to a given interface.
+ *
+ * @param[in] pxInterface: The interface for which a link-local address is looked up.
+ * @param[out] pxIPAddress: The IP address will be copied to this parameter.
+ *
+ * @return pdPASS in case a link-local address was found, otherwise pdFAIL.
+ */
+    static BaseType_t xGetlinklocaladdress( NetworkInterface_t * pxInterface,
```

Comment:
```suggestion
    static BaseType_t xGetLinkLocalAddress( NetworkInterface_t * pxInterface,
```
To make it similar to other function signatures.